.github/workflows/push-image.yml

```
@@ -0,0 +1,42 @@
+# This workflow builds and pushes the expertise image to the Artifact Registry
+
+name: dev-deployment
+
+# Controls when the workflow will run
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+  # Triggers the workflow on push events but only for the master branch
+  push:
+    branches: [ feature/containerize ]
```

Should this be master?
```suggestion
    branches: [ master ]
```